types/api/v2/cluster_types.go

```diff
@@ -75,13 +75,13 @@ func (in *Cluster) GetMasterIPList() []net.IP {
 	return in.GetIPSByRole(common.MASTER)
 }
 
-func (in *Cluster) GetMasterIPStrList() []string {
+func (in *Cluster) GetMasterIPStrList() (ipStrList []string) {
```

Comment:
I think the modification is useless. The effect of original and modified code are the same. @kakaZhou719

Reply:
make will add default value "" to ipStrList, after element  append, the result looks like:  `["","172.16.11.8"]`

Reply:
No, I do not think make will add default value "" to ipStrList.  A unit test case is the best way to show the truth. @kakaZhou719